kubernetes/cray-istio/values.yaml

```diff
@@ -35,6 +35,15 @@ certificate:
     - "*.vshasta.io"
   secretName: ingress-gateway-cert
 
+mqttCertificate:
+    commonName: "dvs-mqtt.local"
```

Comment:
@kburns-hpe  see lint failure ... may not be an issue, but could be on deploy (?)

Reply:
wouldn't affect deploy. I thought I fixed this on all the branches. I'll push out a fix for this and the dates.